test/snapshots/mac/macPackagerTest.js.snap

```diff
@@ -766,6 +766,14 @@ Object {
               "files": Object {
                 "Release": Object {
                   "files": Object {
+                    "node-addon-api": Object {
+                      "files": Object {
+                        "node_addon_api_except.stamp": Object {
+                          "size": "<size>",
+                          "unpacked": true,
+                        },
+                      },
+                    },
```

Comment:
Do you know why these are being added now? I noticed this in my electron/asar PR as well, but could only replicate it on local builds, not on CI builds

Reply:
This PR used yarn to reinstall for this test case. Previously, the package manager on CI was using electron-builder's pnpm by default to reinstall. The difference between these two might be the cause of the issue.